### PR TITLE
App renaming script

### PR DESCRIPTION
**Pull request by chadrien**
PR intending to add support for feature requested in #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ Read the **[Deployment guide](docs/DEPLOYMENT.md)** to learn how to deploy the a
 
 ## Contributing
 
-If you find any problems, please [open an issue](https://github.com/futurice/react-native-kindling/issues/new) or submit a fix as a pull request.
+If you find any problems, please [open an issue](https://github.com/futurice/react-native-pepperoniapptemplate/issues/new) or submit a fix as a pull request.
```
Comment (krivachy) [inline README.md]: Oops! This points to a dead-link. Should be: `https://github.com/futurice/pepperoni-app-kit/issues/new`
An oversight from our side.

Comment (chadrien) [inline README.md]: Yeah I saw it to when reviewing the code, fixed it 😄

**Comment by jevakallio** [comment]
Thank you @chadrien! I won't be on my computer for a couple of days, but in the meantime @krivachy and @tinojunge could test and merge if everything seems to work.

**Comment by jevakallio** [comment]
Eh, that's @tino-junge

**Comment by krivachy** [comment]
Is this the expected output?

```
FL761:pepperoni-app-kit akri$ ./support/rename.sh MonkeyApp
sed: 1: "./android/app/app.iml": invalid command code .
sed: 1: "./android/app/src/main/ ...": invalid command code .
sed: 1: "./android/app/src/main/ ...": invalid command code .
sed: 1: "./android/settings.gradle": invalid command code .
sed: 1: "./bitrise/bitrise.ios.yml": invalid command code .
sed: 1: "./index.android.js": invalid command code .
sed: 1: "./index.ios.js": invalid command code .
sed: 1: "./ios/PepperoniAppTempl ...": invalid command code .
sed: 1: "./ios/PepperoniAppTempl ...": invalid command code .
sed: 1: "./ios/PepperoniAppTempl ...": invalid command code .
sed: 1: "./ios/PepperoniAppTempl ...": invalid command code .
sed: 1: "./ios/PepperoniAppTempl ...": invalid command code .
sed: 1: "./ios/PepperoniAppTempl ...": invalid command code .
sed: 1: "./package.json": invalid command code .
sed: 1: "./src/utils/snapshot.js": invalid command code .
sed: 1: "./android/app/BUCK": invalid command code .
sed: 1: "./android/app/build.gradle": invalid command code .
sed: 1: "./android/app/src/main/ ...": invalid command code .
sed: 1: "./android/app/src/main/ ...": invalid command code .
```

It seemed to have moved the folders correctly, but misses the file contents.

**Comment by krivachy** [comment]
@chadrien forgot the environment details:

OSX, using iTerm:
```
$ bash -version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin15)
Copyright (C) 2007 Free Software Foundation, Inc.
```

**Comment by chadrien** [comment]
Absolutely not. I'll check that.

**Comment by chadrien** [comment]
@krivachy Oh I just pushed a small fix, but actually reading your issue I'm pretty sure that's because your `sed` is the one provided by OSX and not GNU one (precisely the `-i` option is the one causing problem).

Could we declare say that (at least for now) the renaming script requires `gnu-sed`? And add a doc for installation on OSX: `brew install gnu-sed --with-default-names`

**Comment by jevakallio** [comment]
@chadrien I think we need a solution that works on a vanilla OSX box, because that is the majority environment for our users. Having a requirement for custom `sed` would likely cause too much confusion and unnecessary trouble.

We could perhaps write the rename script as an executable node.js file?

**Comment by chadrien** [comment]
@jevakallio It's ok, I found a solution that works for both BSD (Mac) and GNU (linux) sed.

I'll commit it once I successfully ran the usual `run-ios` and `run-android` locally.

**Comment by chadrien** [comment]
@jevakallio  We're good! 👍

**Comment by jevakallio** [comment]
@chadrien excellent! I'll try to get a couple of other PRs merged in first to avoid unnecessary rename conflicts, will be pulling this in shortly.

**Comment by jevakallio** [comment]
@chadrien I've merged your commits to master as part of https://github.com/futurice/pepperoni-app-kit/pull/38. Upgrading Pepperoni to React Native 0.26 and CocoaPods 1.0 caused quite a mess, so I had to rebase a lot of the changes manually.

I found and fixed one missing rename step https://github.com/futurice/pepperoni-app-kit/pull/38/commits/8cb03c402a8062e7bcbfa5a0a71f38d1377a54d3, and added some instructions to run a clean build after rename, as my app would not build without it after the rename.

Thanks for the contribution! 🍕

**Comment by chadrien** [comment]
🎉

**Pull request closed without merging**